Application/Common/Models/RemoveFromShortlistModel.cs

```diff
@@ -0,0 +1,6 @@
+namespace Application.Common.Models;
+
+public class RemoveFromShortlistModel : ShortlistModelCommon
```

Comment:
Are these models only used for the tests? If so it might be worth moving them to the Tests namespace?

Reply:
Please see my response above.